src/conversion/model-to-view-converters.js

```diff
@@ -351,11 +351,16 @@ export function wrapRange( elementCreator ) {
 			return;
 		}
 
+		const viewRange = conversionApi.mapper.toViewRange( data.range );
+
+		if ( viewRange.isCollapsed ) {
+			return;
+		}
+
```

Comment:
Yea, it was my first idea to fix it. But then I realized that nothing should happen if I do not return because https://github.com/ckeditor/ckeditor5-engine/blob/master/src/view/writer.js#L517-L519 This code for sure should be moved before `validateRangeContainer`, but when I did it new bugs appeared.